### PR TITLE
docs(alert): fixes

### LINKED ISSUE #1027: [docs]: alert edits

### PR DESCRIPTION
**Pull request by eyevana**
Closes #1027

## What I did

1. Minor updates to the alert docs

## Testing Instructions

1. Verify the changes against the issues outlined in #1027.

### CODE REVIEW

File: .changeset/rh-alert-variant.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+`<rh-alert>`: clarify that the `variant` attribute takes values: `alternate`,
```
Comment (bennypowers) [inline .changeset/rh-alert-variant.md:4]: Gonna approve for now, but @zeroedin can you give this the once-over?

**Comment by eyevana** [comment]
@coreyvickery ready! :) (I also included the patch to the Pagination Style section re: wonky interaction state tables in this PR)

File: elements/rh-alert/demo/alternate.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+</rh-context-provider>
+
+<script type="module">
+  import '@rhds/elements/rh-alert/rh-alert.js';
+  import '@rhds/elements/lib/elements/rh-context-provider.js';
```
Comment (zeroedin) [inline elements/rh-alert/demo/alternate.html:47]:
```Suggestion
  import '@rhds/elements/rh-context-provider/rh-context-provider.js';
```

Sub page demo's are broken on the DP.  I have a feeling this is preprep for https://github.com/RedHat-UX/red-hat-design-system/pull/996 but should we wait to fix after that merges so we aren't pushing out broken demos?

**Review by zeroedin — APPROVED** [review]
Just the one comment about the demo's, otherwise LGTM

**Review by coreyvickery — APPROVED** [review]
All good.

File: elements/rh-alert/demo/alternate.html (hunk 2 above)
Comment (bennypowers) [reply elements/rh-alert/demo/alternate.html:47]: yeah might as well wait on that pr

**Pull request merged**